### ISSUE TITLE
Fecha de Aire (en la que se emitió el cap. de las Series)

### ISSUE THREAD

**Issue by nicolasraffaele** [Cuevana/storm#82]
No es un problema pero si se empieza a agregar en la API de Cuevana el dato de fecha de emisión yo lo agrego a la info de las series... ¿que les parece?

**Comment by Cuevana** [comment]
Actualmente no recolectamos ese dato. Sólo tenemos el de fecha de indexado a la base de datos, lo cual puede funcionar como reemplazo por ahora, aunque casi siempre es el día siguiente al de emisión.
Para obtener los días correctos deberíamos usar otra base de datos de series.

**Comment by nicolasraffaele** [comment]
Claro, me iba a fijar en la api de popcorn pero no puedo entrar el git, de ultima es verdad se puede poner la fecha de indexado por el momento no es mala idea. Ah tambien mostrar el genero de la serie en los capitulos no solo en la info general, y como propuse el cambio de lugar de Reproducir en el lugar de Ir a serie y pasar el Ir a serie al de Reproducir, yo no toco esas cosas sin permiso previo. Saludos
